src/scss/config/scale/_font-sizes.scss

```diff
@@ -5,6 +5,7 @@ $xxsmall: 0.6rem;
 $medium: 1.25rem;
 $large: calc(1.25rem + 0.5vw);
 $xlarge: calc(1.75rem + 1vw);
+$xxlarge: calc(2.75rem + 1.25vw);
```

Comment:
Is this used somewhere?

Reply:
removed in https://github.com/oddbird/oddleventy/pull/859/commits/2f0a8ac174fcac4a6b218bc025e5cb720836f8c6